src/llama.cpp

```diff
@@ -2151,6 +2216,7 @@ enum e_model {
     MODEL_1B,
     MODEL_1_3B,
     MODEL_1_4B,
+    MODEL_1_6B,
```

Comment:
The string representation of that size should be handled too

```diff
diff --git a/src/llama.cpp b/src/llama.cpp
index 96725500..1aadab4c 100644
--- a/src/llama.cpp
+++ b/src/llama.cpp
@@ -5161,6 +5161,7 @@ static const char * llama_model_type_name(e_model type) {
         case MODEL_1B:            return "1B";
         case MODEL_1_3B:          return "1.3B";
         case MODEL_1_4B:          return "1.4B";
+        case MODEL_1_6B:          return "1.6B";
         case MODEL_2B:            return "2B";
         case MODEL_2_8B:          return "2.8B";
         case MODEL_3B:            return "3B";
```

Reply:
Yes XD. I missed that one